cmd/check-gh-automation/main_test.go

```diff
@@ -239,7 +284,11 @@ func TestCheckRepos(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			failing, err := checkRepos(tc.repos, tc.bots, "openshift-ci", tc.ignore, tc.mode, client, logrus.NewEntry(logrus.New()), newFakePluginConfigAgent(), newFakeProwConfigAgent().Config().Tide.Queries.QueryMap())
+			// Decide whether to enforce merge-bot admin based on the test case
+			enforceMergeBotAdmin := len(tc.adminBots) > 0 && tc.adminBots[0] == "openshift-merge-robot"
```

Comment:
Rather than having this logic here, lets just always check for admin access for `openshift-merge-robot`. You will want a test case where the fake client returns that it is an admin, and another where it does not. You can do this by having the test cases use different repos.